src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -492,7 +492,7 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   )
 
   const handleBackClick = useCallback(() => {
-    browserNavigate('/pools')
+    browserNavigate(-1)
```

Comment:
If we want to be fancy, probably want to check if there *is* -1 here

Reply:
https://github.com/shapeshift/web/pull/10939#pullrequestreview-3396920663

I guess, we don't care then! I don't remember if we have a route coming from somewhere else but there would be a -1 even in this case

Direct access maybe would be an issue but hey, that's quite nitpick/edge case